content/en/blog/falco-0-34-0/index.md

```diff
@@ -0,0 +1,122 @@
+
+Dear community, today we are delighted to announce the release of Falco 0.34.0  🎉!
+
+A big thank you to the community for helping get the latest release out. The Falco community is thrilled about this release and cannot wait to share the latest goodies. Check out the newest features from this most recent launch to learn more ⬇️. To read a more detailed account of the release, see [v0.34.0 in the changelog]().
+
+## What’s new? 🆕
+In this release we saw more than xxx pull requests across the repos of Falco and its libraries. Thank you to our maintainers and contributors, as this would not happen without your support, dedication, and contribution.
+
+## Updates - TL;DR 🩳
+In release v0.34.0 the community focused on addressing the following key features: 
+- Downloading and dynamically updating Falco rules at runtime
+- Shipping the brand new experimental modern eBPF probe
+- Designing more ways to catch suspicious executions
+
+## Automatic rules update 🔄
+A few questions that often come up when using Falco is how can we update rules once Falco is installed in the cluster and, how do we get updated rules from the Falco organization without having to wait for the next release? This is the first release to include [falcoctl](https://github.com/falcosecurity/falcoctl) with an out of the box solution to do exactly that!
+
+When using the new Falco Helm Chart 3.0.0 rules will automatically be updated from the official repository. To learn more about this feature and how to configure it read [xxx] links. Likewise, when using a SystemD based install you can ([xxx] link) configure Falco to automatically update rules.
+
+## Modern eBPF probe 👨‍🚀
+Last quarter, Andrea published the blog, “[Getting started with modern BPF probe in Falco](https://falco.org/blog/falco-modern-bpf/),” and announced that the new experimental eBPF probe had landed among us bringing with it a few key features: [CO-RE paradigm](https://falco.org/blog/falco-modern-bpf/#co-re-paradigm), [BPF Ring Buffer map](https://falco.org/blog/falco-modern-bpf/#bpf-ring-buffer-map), [BTF-enabled program](https://falco.org/blog/falco-modern-bpf/#btf-enabled-program), [BPF global variables](https://falco.org/blog/falco-modern-bpf/#bpf-global-variables), [BPF skeleton](https://falco.org/blog/falco-modern-bpf/#btf-enabled-program), and finally [Multi-arch support](https://falco.org/blog/falco-modern-bpf/#multi-arch-support). 
+
+### Why a new probe? 👽
+The old probe supported old kernels (>=4.14) that can not take advantage of the new shiny eBPF features. While it would be great to have only one probe that works for every kernel version, recent features change (and simplify!) the way we write, maintain and deploy the code so deeply that a new fresh probe is the most reasonable solution. In order to leverage these recent eBPF improvements and use the new probe you will need a kernel version >= 5.8.
+
+### Modern eBPF in action 🎬
+Try it now by following these instructions [xxx link]
```

Comment:
* link to the documentation section https://github.com/falcosecurity/falco-website/pull/800